### ISSUE TITLE
Build fails without gobject-introspection

### ISSUE THREAD

**Issue by gber** [AyatanaIndicators/libayatana-appindicator-glib#28]
If gobject-introspection is not installed the build fails due to missing binaries. gobject-introspection should either be a mandatory build dependency or building introspection files should be skipped if it isn't installed.

**Comment by gber** [comment]
This happens due to `find_package(GObjectIntrospection REQUIRED QUIET)`, "QUIET" is the opposite of "REQUIRED" and tells CMake to ignore the missing package and continue. I can fix this and some other bugs in another PR once PR #17 is finished.
